docs/SUMMARY.md

```diff
@@ -13,7 +13,7 @@
   * [Sub States](advanced/sub-states.md)
   * [Cancelation](advanced/cancelation.md)
   * [Caching](advanced/cache.md)
-  * [Pub Sub](advanced/pubsub.md)
+  * [Action Handlers](advanced/action-handlers.md)
```

Comment:
are we removing any docs on the action stream?

Reply:
No, just name update.